.github/workflows/python-app.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: gitchangelog
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Git repository creation (required)
+      run: |
+        git init
+        git fetch
+        git reset origin/main
+        git checkout origin/main
```

Comment:
As noted in Slack, this sucks, but in order to run tests it needs a real repository. 

It just occurred to me that the repo is already checked out, could it be an issue with wrong path that `autogen.sh` doesn't understand somehow?

Reply:
> It just occurred to me that the repo is already checked out, could it be an issue with wrong path that `autogen.sh` doesn't understand somehow?

Yes, i think it is. We should probably apply proposed modernization PR on main project: https://github.com/vaab/gitchangelog/pull/134 which beside other improvements also removes `autogen.sh`.